pom.xml

```diff
@@ -127,6 +130,16 @@
                 <artifactId>mockito-junit-jupiter</artifactId>
                 <version>${mockito.version}</version>
             </dependency>
+            <dependency>
+                <groupId>com.icegreen</groupId>
+                <artifactId>greenmail</artifactId>
+                <version>1.5.7</version>
```

Comment:
Might pull this into a property as well

Reply:
Line 141 could also be extracted to a property.